documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1047 @@
+ifdef::ostack[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openstack
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  domainName: <domain_name>
+  projectName: <project_name>
+  regionName: <region name> <5>
+  cacert: | <6>
+    <ca_certificate>
+  url: <api_end_point> <7>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {osp} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
+<5> Specify the name of the {osp} region.
+<6> Enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`. Note that if you set `insecureSkipVerify` to `false` and you do not set  `cacert`, {project-short} attempts to use the system CA.
+<7> Specify the API end point URL, for example, `https://<identity_service>/v3`.
+endif::[]
+
+ifdef::cnv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openshift
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  url: <api_end_point> <5>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {virt} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
+<5> Specify the API end point URL.
+endif::[]
++
+
+. Create a `Provider` manifest for the source provider:
++
+ifdef::vmware[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: vmware
+  url: <api_end_point> <1>
+  settings:
+    vddkInitImage: <registry_route_or_server_path>/vddk:<tag> <2>
```

Comment:
in 2.6 we add to the `settings` also the `sdkEndpoint` key that can be set with vcenter or esxi